graph/types/blockchain.go

```diff
@@ -61,3 +62,18 @@ func MarshalInt(x int) graphql.Marshaler {
 
 // Is this going to work?
 var UnmarshalInt = graphql.UnmarshalInt
+
+func MarshalBigInt(x *big.Int) graphql.Marshaler {
+	return graphql.WriterFunc(func(w io.Writer) {
+		_, _ = io.WriteString(w, strconv.Quote(x.String()))
+	})
+}
+
+func UnmarshalBigInt(v interface{}) (*big.Int, error) {
+	s, ok := v.(*big.Int)
```

Comment:
We should do an end to end test. I think this is going to fail like the `Bytes` stuff failed.